tests/test_geometamaker.py

```diff
@@ -567,6 +572,24 @@ def test_preexisting_metadata_document_new_profile(self):
         self.assertEqual(
             new_resource.contact.individual_name, 'bob')
 
+    def test_preexisting_metadata_profile_not_overwritten(self):
+        """Test doc with contact info is not overwritten by blank profile."""
+        import geometamaker
+
+        datasource_path = os.path.join(self.workspace_dir, 'raster.tif')
+        create_raster(numpy.int16, datasource_path)
+        resource = geometamaker.describe(datasource_path)
+        resource.set_contact(individual_name='alice')
+        resource.write()
+
+        # mocking should mean there is no config, but just to be certain
+        config = geometamaker.Config()
+        config.delete()
+
+        new_resource = geometamaker.describe(datasource_path)
+        self.assertEqual(
+            new_resource.contact.individual_name, 'alice')
+
```

Comment:
This test doesn't pertain to any changes in this PR. I was just second-guessing this behavior and thought I would write a test to confirm how it works.